services/frontend/src/components/profileForms/langProficiencyForm/LangProficiencyFormView.jsx

```diff
@@ -293,6 +293,7 @@ const LangProficiencyFormView = ({
         <Form.Item
           name={statusName}
           label={<FormattedMessage id="lang.status" />}
+          rules={[Rules.required]}
```

Comment:
can you add aria-required="true" to the input for accessibility. You can go to PrimaryInfoFormView.jsx for an example

Reply:
is aria-required only for input tags ? I was wondering because I cant find it used in Select tags https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute

Reply:
I used it it on the primary info form and i tested with the screen reader and it is working. we just need the screen reader to pick up the "is required"